src/scraping/start_for_life/scrape_startforlife.py

```diff
@@ -0,0 +1,187 @@
+    removed_indices = []
+    # Keep track of the last non-empty header index
+    last_non_empty_index = -1
+
+    for i in range(len(headers)):
+        # If header is non-empty
+        if headers[i]:
+            new_headers.append(headers[i])
+            new_sections.append(sections[i])
+            last_non_empty_index = len(new_headers) - 1
+        # If header is empty
+        else:
+            # Add index to removed list
+            removed_indices.append(i)
+            # If there was a previous non-empty header
+            if last_non_empty_index != -1:
+                new_sections[last_non_empty_index] += sections[i]
+
+    return new_headers, new_sections, removed_indices
+
+
+def web_scraper(url: str, timeout: float = 10) -> (List[str], List[str], List[str]):
+    """Scrape a web page and return the content
+
+    Args:
+        url
+            URL of the web page
+
+        timeout
+            Timeout in seconds for the request to the website
+
+    Returns:
+        Tuple of lists with sections' headers, content under each header and the sections' classes
+    """
+    # Fetch webpage
+    response = requests.get(
+        url,
+        timeout=timeout,
+        headers=REQUEST_HEADERS,
+    )
+
+    # Parse with BeautifulSoup
+    soup = BeautifulSoup(response.text, "html.parser")
+
+    # Get all divs with the specified list of class names
+    divs = soup.find_all("div", class_=DIV_CLASS)
+
+    # Get all header (h1, h2 etc) tags from each of the divs
+    headers = [[d.get_text() for d in div.find_all(HEADER_TAGS)] for div in divs]
+
+    # Process the div content: remove headers (h1, h2, etc)
+    divs_processed = []
+    for div in divs:
+        div_processed = div
+        for header in div_processed.find_all(HEADER_TAGS):
+            header.decompose()
+        divs_processed.append(div_processed)
+    # Get class names of the divs
+    divs_classes = [div.get("class") for div in divs]
+    # Keep only text
+    divs_processed = [div.get_text() for div in divs_processed]
+    # Merge divs without header into previous div
+    headers, divs_processed, removed_indices = merge_sections_based_on_headers(headers, divs_processed)
+    # Remove corresponding elements from divs_classes
+    divs_classes = [div for i, div in enumerate(divs_classes) if i not in removed_indices]
+    # Remove new lines from the beginning and end of each div
+    divs_processed = [div.strip() for div in divs_processed]
+    # and replace multiple new lines with a space
+    divs_processed = [div.strip().replace("\n", " ") for div in divs_processed]
+    return headers, divs_processed, divs_classes
+
+
+def has_multiple_items(items: List[str]) -> bool:
+    """Check if there are multiple headers in the second-level list"""
+    for item in items:
+        if len(item) > 1:
+            logging.info(f"Multiple headers in {item}")
+            return False
+        else:
+            return True
+
+
+if __name__ == "__main__":
```

Comment:
`scrape_sitemap` and `scrape_startforlife` could be one script without needing to store the sitemap's URLs in a CSV